library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +68,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    private Map<String,DayViewDecorator> dayViewDecorators;
```

Comment:
May be the easiest way to do that is to have something like
```
calendar.invalidateDecorators();
```
In any case, we would have to redraw them all